lib/src/line_numbers/line_number_controller.dart

```diff
@@ -13,8 +13,8 @@ class LineNumberController extends TextEditingController {
     TextStyle? style,
     bool? withComposing,
   }) {
-    final children = <TextSpan>[];
-    final list = text.split("\n");
+    final children = <InlineSpan>[];
```

Comment:
I am experimenting with adding other spans here locally. This simplifies merges to me.